src/main/java/io/strimzi/kafka/quotas/StaticQuotaConfig.java

```diff
@@ -62,6 +64,10 @@ long getSoftStorageQuota() {
         return getLong(STORAGE_QUOTA_SOFT_PROP);
     }
 
+    String getStoragePercents() {
```

Comment:
Percentages (percents isn't really an English word, at least not one in common use).